### PR TITLE
updateOrderStatus connect backend

### PR DESCRIPTION
**Pull request by OuChiaYun**
http://localhost:5173/order/updateOrderStatus

預設shop id=1

### CODE REVIEW

**Comment by nonorze-VOXX** [comment]
conflict

**Comment by nonorze-VOXX** [comment]
yo bro your goto look go wrong page

**Comment by OuChiaYun** [comment]
> conflict

 fixed

**Comment by OuChiaYun** [comment]
> yo bro your goto look go wrong page

目前先 ./，等history page 結束再改

File: frontend/src/lib/components/PUA/orderStatusCard.svelte [hunk 1]
```diff
@@ -5,17 +5,44 @@
...
 </script>
 
 <div class=" w-3/7">
-	<div class="flex flex-wrap items-center justify-center gap-8">
-		<button class="flex items-center gap-5 rounded-xl bg-white p-4 shadow-xl">
+	<div class=" flex flex-wrap items-center justify-center gap-8">
+		<a
+			{href}
+			class=" group flex items-center gap-5 rounded-xl bg-white p-4 shadow transition-all duration-300 hover:-translate-y-6 hover:transform hover:shadow-2xl hover:shadow-zinc-600"
+		>
```
Comment (nonorze-VOXX) [inline frontend/src/lib/components/PUA/orderStatusCard.svelte]: press button will trigger it

File: frontend/src/routes/order/updateOrderStatus/+page.svelte [hunk 2]
```diff
@@ -7,27 +11,95 @@
...
+				text: icon_array[orderResp.status - 1].text,
+				user: orderResp.user_name,
+				status: orderResp.status
+			};
+		} else {
+			statusCardContent = statusCardContent.filter((_, i) => i !== index);
+		}
+
+		return;
+	}
```
Comment (nonorze-VOXX) [inline frontend/src/routes/order/updateOrderStatus/+page.svelte]: use invalidateAll to refresh page

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]

File: frontend/src/lib/components/PUA/orderStatusCard.svelte (hunk 1 above)
Comment (OuChiaYun) [reply frontend/src/lib/components/PUA/orderStatusCard.svelte]: fixed

File: frontend/src/routes/order/updateOrderStatus/+page.svelte (hunk 2 above)
Comment (OuChiaYun) [reply frontend/src/routes/order/updateOrderStatus/+page.svelte]: fixed

File: frontend/src/lib/components/PUA/orderStatusCard.svelte [hunk 3]
```diff
@@ -33,11 +64,12 @@
...
 
-			<div
+			<button
+				on:click|stopPropagation
 				class=" flex h-20 w-40 items-center justify-center rounded-2xl bg-PUA-stone p-2 text-center font-bold leading-relaxed text-white hover:border-[3px] hover:border-PUA-stone hover:bg-white hover:text-PUA-stone"
 			>
 				{statusCardContent.text}
-			</div>
+			</button>
 		</button>
```
Comment (nonorze-VOXX) [inline frontend/src/lib/components/PUA/orderStatusCard.svelte:73]: 之前不是說按鈕在按鈕裡面會有問題嗎

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]

File: frontend/src/lib/components/PUA/orderStatusCard.svelte (hunk 3 above)
Comment (OuChiaYun) [reply frontend/src/lib/components/PUA/orderStatusCard.svelte:73]: 不是 是按鈕疊在按鈕上面，中間有div 隔著好像會有問題

**Review by nonorze-VOXX — APPROVED** [review]
LGTM

**Pull request merged**